pulsar-common/src/main/java/org/apache/pulsar/common/protocol/Commands.java

```diff
@@ -368,7 +368,7 @@ public static BaseCommand newErrorCommand(long requestId, ServerError serverErro
         cmd.setError()
                 .setRequestId(requestId)
                 .setError(serverError)
-                .setMessage(message);
+                .setMessage(message != null ? message : "");
```

Comment:
I agree with @mattisonchao. In my opinion, it seems like we should always set a message here. Do we have a follow up PR to address the case where the message was `null`?